src/Point.h

```diff
@@ -49,9 +48,10 @@ class Point {
  * @param x x-coordinate
  * @param y y-coordinate
  */
-inline void Point::setCoords(const double x, const double y) {
-  _x = x;
-  _y = y;
+inline void Point::setCoords(const double x, const double y, const double z) {
+  _xyz[0] = x;
+  _xyz[1] = y;
+  _xyz[2] = z;
```

Comment:
I personally like this notation better, it follows with what is used in OpenMC, but recognize that it is just a personal preference issue (I don't see any performance issues either way).